app/infra/repositories/statistics/base.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from uuid import uuid4
+
+
+@dataclass
+class QuoteObject:
+    quote: str
+    author: str
+    send_dt: int
+    picture_url: str
+    picture_name: str
+    index: str = field(default_factory=lambda: str(uuid4()))
+
+
+@dataclass
+class BaseStatisticsRepository(ABC):
```

Comment:
Why do you require this?

Reply:
I'm using this for dependency inversion.